app/_components/TestComponent/BadComponent.tsx

```
@@ -0,0 +1,93 @@
+    { id: 2, name: "Item 2" },
+    { id: 3, name: "Item 3" },
+];
+
+// This class component shouldn't be mixed with functional components in one file
+class OldComponent extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { count: 0 };
+    }
+
+    componentWillMount() { // Deprecated method
```

ESLint

[error] componentWillMount is deprecated since React 16.9.0, use UNSAFE_componentWillMount instead, see https://reactjs.org/docs/react-component.html#unsafe_componentwillmount. Use https://github.com/reactjs/react-codemod#rename-unsafe-lifecycles to automatically update your components.

(react/no-deprecated)